docs/en/admins/09_AccessControl.md

```diff
@@ -34,6 +34,23 @@ You may alternatively pass a `TRUSTED_PROXY` environment variable in a format co
 
 > ☠️ WARNING: FreshRSS will trust any IP configured in the `trusted_sources` option, if your proxy isn’t properly secured, an attacker could simply attach this header and get admin access.
 
+### Authentik Proxy Provider
```

Comment:
See also https://github.com/FreshRSS/FreshRSS/blob/edge/docs/en/admins/16_OpenID-Connect-Authentik.md

Reply:
Yeah, I saw it and I couldn't get it to work. I see there's new docs. Would you like me to remove this section?

Reply:
https://github.com/FreshRSS/FreshRSS/pull/5853/commits/ae0630f9c43944c9b80ddb3b4e44082eef51d880